optimize/backend/pom.xml

```diff
@@ -416,6 +462,397 @@
       <artifactId>camunda-license-check</artifactId>
       <version>2.9.0</version>
     </dependency>
+
+    <!-- used/undeclared -->
+    <dependency>
+      <groupId>org.mock-server</groupId>
+      <artifactId>mockserver-core</artifactId>
+      <version>5.15.0</version>
```

Comment:
I think this, and many of the ones below, already have properties defined for their version. I know this is WIP but we should use them where possible instead of having their version here